catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/FilterReader.java

```diff
@@ -28,15 +31,20 @@ public FilterReader() throws JAXBException {
     this.context = JAXBContext.newInstance(FilterType.class);
   }
 
-  public JAXBElement<FilterType> unmarshalFilter(InputStream inputStream) throws JAXBException {
+  public JAXBElement<FilterType> unmarshalFilter(InputStream inputStream)
+      throws JAXBException, XMLStreamException {
     return unmarshal(inputStream, FilterType.class);
   }
 
   @SuppressWarnings("unchecked")
   private <T> JAXBElement<T> unmarshal(InputStream inputStream, Class<T> tClass)
-      throws JAXBException {
+      throws JAXBException, XMLStreamException {
+    XMLInputFactory xif = XMLInputFactory.newFactory();
```

Comment:
❗️ We should be using XMLUtils here, and it's also worth noting that we've had a PR open to fix this for a solid chunk of time now https://github.com/codice/ddf/pull/3121, so it might be worth getting that one closed and then just dropping these changes